src/applications/gi/sass/partials/_gi-search-page.scss

```diff
@@ -147,5 +149,21 @@
       }
     }
 
+    .preferred-flag {
+      text-align: right;
+      border-radius: 3px 3px 3px 3px;
+      font-size: 0.9em;
+      font-weight: bold;
+      padding-top: 1em;
+      padding-bottom: .25em;
+      padding-right: 1.5em;
+      margin: 1em 0 0 1em;
+
+      i.fa {
+        color: $color-gold;
```

Comment:
Might not need this if you're already specifying the color through the class, I think.

Reply:
Removed unnecessary styling.